automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala

```diff
@@ -130,7 +128,7 @@ class SamApiSpec extends AnyFreeSpec with BillingFixtures with Matchers with Sca
       val userStatus = Sam.user.status()(userAuthToken).get
 
       // user a brand new billing project to ensure known state for pet (not present)
-      withBrandNewBillingProject("new-pet-test") { projectName =>
+      withCleanBillingProject(owner, userEmails = List(anyUser.email)) { projectName =>
```

Comment:
the comment above mentions why we were using a brand new billing project intentionally. Since we've changed that now, we can probably remove the comment

Reply:
since this change may cause other errors (which the comment alludes to), maybe we should leave a comment here as a breadcrumb in case it does cause issues?

On the other hand, I'm also ok with not leaving a comment since we can probably just look at the file history / git blame if/when we need to debug this test. 

On the other other hand, while leaving a comment would add some clutter, it would also save us time/effort in the future if we need to look at the test again and so I think I'm leaning toward adding a comment. 

Any thoughts?

Reply:
I'm not sure if this even passes yet! I'll remove the comment and update it when it passes!